db-engine/src/main/java/com/splicemachine/db/impl/sql/catalog/SYSALIASESRowFactory.java

```diff
@@ -48,346 +48,338 @@
-		boolean				systemAlias = false;
-		AliasInfo			aliasInfo = null;
-
-		/* 1st column is ALIASID (UUID - char(36)) */
-		col = row.getColumn(SYSALIASES_ALIASID);
-		aliasID = col.getString();
-		aliasUUID = getUUIDFactory().recreateUUID(aliasID);
-
-		/* 2nd column is ALIAS (varchar(128)) */
-		col = row.getColumn(SYSALIASES_ALIAS);
-		aliasName = col.getString();
-
-		/* 3rd column is SCHEMAID (UUID - char(36)) */
-		col = row.getColumn(SYSALIASES_SCHEMAID);
-		UUID schemaUUID = col.isNull() ? null : getUUIDFactory().recreateUUID(col.getString());
-
-		/* 4th column is JAVACLASSNAME (longvarchar) */
-		col = row.getColumn(SYSALIASES_JAVACLASSNAME);
-		javaClassName = col.getString();
-
-		/* 5th column is ALIASTYPE (char(1)) */
-		col = row.getColumn(SYSALIASES_ALIASTYPE);
-		sAliasType = col.getString();
-		if (SanityManager.DEBUG)
-		{
-			SanityManager.ASSERT(sAliasType.length() == 1, 
-				"Fifth column (aliastype) type incorrect");
-			switch (sAliasType.charAt(0))
-			{
-				case AliasInfo.ALIAS_TYPE_PROCEDURE_AS_CHAR: 
-				case AliasInfo.ALIAS_TYPE_FUNCTION_AS_CHAR: 
-				case AliasInfo.ALIAS_TYPE_SYNONYM_AS_CHAR: 
-				case AliasInfo.ALIAS_TYPE_UDT_AS_CHAR: 
-				case AliasInfo.ALIAS_TYPE_AGGREGATE_AS_CHAR:
-					break;
-
-				default: 
-					SanityManager.THROWASSERT("Invalid type value '"
-							+sAliasType+ "' for  alias");
-			}
-		}
-
-		cAliasType = sAliasType.charAt(0);
-
-		/* 6th column is NAMESPACE (char(1)) */
-		col = row.getColumn(SYSALIASES_NAMESPACE);
-		sNameSpace = col.getString();
-		if (SanityManager.DEBUG)
-		{
-			SanityManager.ASSERT(sNameSpace.length() == 1, 
-				"Sixth column (namespace) type incorrect");
-			switch (sNameSpace.charAt(0))
-			{
-				case AliasInfo.ALIAS_NAME_SPACE_PROCEDURE_AS_CHAR: 
-				case AliasInfo.ALIAS_NAME_SPACE_FUNCTION_AS_CHAR: 
-				case AliasInfo.ALIAS_TYPE_SYNONYM_AS_CHAR: 
-				case AliasInfo.ALIAS_TYPE_UDT_AS_CHAR: 
-				case AliasInfo.ALIAS_TYPE_AGGREGATE_AS_CHAR:
-					break;
-
-				default: 
-					SanityManager.THROWASSERT("Invalid type value '"
-							+sNameSpace+ "' for  alias");
-			}
-		}
-
-		cNameSpace = sNameSpace.charAt(0);
-
-
-		/* 7th column is SYSTEMALIAS (boolean) */
-		col = row.getColumn(SYSALIASES_SYSTEMALIAS);
-		systemAlias = col.getBoolean();
-
-		/* 8th column is ALIASINFO (com.splicemachine.db.catalog.AliasInfo) */
-		col = row.getColumn(SYSALIASES_ALIASINFO);
-		aliasInfo = (AliasInfo) col.getObject();
-
-		/* 9th column is specific name */
-		col = row.getColumn(SYSALIASES_SPECIFIC_NAME);
-		String specificName = col.getString();
-
-
-		/* now build and return the descriptor */
-		return new AliasDescriptor(dd, aliasUUID, aliasName,
-										schemaUUID, javaClassName, cAliasType,
-										cNameSpace, systemAlias,
-										aliasInfo, specificName);
-	}
+public class SYSALIASESRowFactory extends CatalogRowFactory {
+    public static final String TABLENAME_STRING = "SYSALIASES";
+    public static final int SYSALIASES_ALIASINFO = 8;
```

Comment:
Move this down with other column names for better readability

Reply:
done.